AdaSupport/embed
# INT-319: Introduce Drag and Drop to Embed + ConnectorURL Fix

### What
* Introduce `dragAndDrop` option to embed
* If turned on Button & Intros will be draggable
* Button/Intro will not active during drag
* Added cluster variable to connector's url
### Test
* Load embed and drag it around
* Open & Close will no longer make the button disappear
---
